src/components/SortingOptions.jsx

```diff
@@ -1,35 +1,27 @@
 import React, { useEffect, useState } from 'react'
 import { getAPI } from '../API'
 
-const SortingOptions = ({category = ""}) => {
+const SortingOptions = () => {
     const [categoryList, setCategoryList] = useState([])
-    console.log("this is the category", category)
     
     useEffect(() => {
         getAPI('categories').then(({categories})=> {
         setCategoryList(categories)
         }) 
     }, [])
 
-    const handleChange = (event) => {
-        window.location.assign(`http://localhost:3000/reviews/${event.target.value}`)
-    }
-
     const categoryElements = categoryList.map(({slug}) => {
         const categoryStr = slug[0].toUpperCase() + slug.slice(1).replace(/-/g, " ")
+        const endpoint = `http://localhost:3000/reviews/category/${slug}`
```

Comment:
Not sure why you have had to use localhost in the endpoint here. Works fine without for me and we can't keep that in production! Drop in an nc help if you continue to have issues